beacon-chain/params/config.go

```diff
@@ -18,3 +22,10 @@ const (
 	// NotariesPerCrosslink fixed to 100.
 	NotariesPerCrosslink = 100
 )
+
+// Web3ServiceConfig defines a config struct for a web3 service to use through its life cycle.
+type Web3ServiceConfig struct {
```

Comment:
I think this should live next to the service itself.

Reply:
What do you mean? which service?